### ISSUE TITLE
Produce an error if @classmethod comes before @asynq()

### ISSUE THREAD

**Issue by JelleZijlstra** [quora/pyanalyze#391]
This code doesn't work properly (`.asynq` isn't accessible):
```python
class Cls:
        @classmethod
        @asynq()
        def f() -> None:
            pass
```

Pyanalyze should detect this and warn about it.
